app/models/study.rb

```diff
@@ -89,6 +89,10 @@ def are_synced
     end
   end
 
+  #User annotations are per study
```

Comment:
We should add delete statements for user_annotations and user_data_arrays into Study.delete_queued_studies.  This will improve study deletion performance.